Jenkinsfile

```diff
@@ -56,7 +56,12 @@ pipeline {
                         cd /home/saluser/repos/ts_idl
                         /home/saluser/.checkout_repo.sh ${work_branches}
                         git pull
+                        cd /home/saluser/repos
+                        rm -rf ts_ess_common  # In case it becomes part of repos
```

Comment:
I strongly recommend removing this line. If ts_ess_common becomes part of `repos` then the build will fail which will be a sign to adjust Jenkinsfile to the new situation. If this line remains then that will go unnoticed.

Reply:
OK.